optimism/src/keccak/interpreter.rs

```diff
@@ -115,7 +115,7 @@ pub trait KeccakInterpreter<F: One + Debug + Zero> {
     fn variable(&self, column: KeccakColumn) -> Self::Variable;
 
     /// Adds one KeccakConstraint to the environment if the selector holds
-    fn constrain(&mut self, tag: Constraint, flag: Self::Variable, x: Self::Variable);
+    fn constrain(&mut self, tag: Constraint, if_true: Self::Variable, x: Self::Variable);
```

Comment:
`if_true` is a variable expressing the selector result? (I see, yeah.) Both names are not exactly directly intuitive IMHO...